src/App.tsx

```diff
@@ -1,15 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title
+      .toLowerCase()
+      .includes(query.toLowerCase().trim())
+    || movie.description
+      .toLowerCase()
+      .includes(query.toLowerCase().trim()),
+  );
```

Comment:
a helper function screams begging to be introduced 🙂 
or at least a query variable
```
const normalizedQuery = query.toLowerCase().trim()
```